src/OpenGL.cpp

```diff
@@ -1327,6 +1327,10 @@ void OGLRender::_initExtensions()
 	m_bImageTexture = false;
 #endif
 	LOG(LOG_VERBOSE, "ImageTexture support: %s\n", m_bImageTexture ? "yes" : "no");
+	if (m_bImageTexture == false)
+	{
+		LOG(LOG_WARNING, "Image Textures not supported. Without OpenGL >= 4.3 or GLES >= 3.1, depth buffer emulation will not work");
```

Comment:
>> depth buffer emulation will not work

it is incorrect. "N64 depth compare" option will not work.
Depth buffer copy to RDRAM does not use Image Textures. Thus, effects based on depth buffer check in RDRAM will work.